src/main.qml

```diff
@@ -83,32 +104,28 @@ Application {
             property real w: (grid.width / columns) - ((spacing * (columns - 1)) / columns)
             property real h: (grid.height / rows) - ((spacing * (rows - 1)) / rows)
 
-            CalcButton { width: grid.w; height: grid.h; operation: "C" }
-            CalcButton { width: grid.w; height: grid.h; operation: CalcEngine.leftArrow }
-            CalcButton { width: grid.w; height: grid.h; operation: CalcEngine.plusminus }
             CalcButton { width: grid.w; height: grid.h; operation: CalcEngine.division; }
-
-
             CalcButton { width: grid.w; height: grid.h; operation: "7"; }
             CalcButton { width: grid.w; height: grid.h; operation: "8"; }
             CalcButton { width: grid.w; height: grid.h; operation: "9"; }
-            CalcButton { width: grid.w; height: grid.h; operation: CalcEngine.multiplication; }
+            CalcButton { width: grid.w; height: grid.h; operation: "-"; }
 
+            CalcButton { width: grid.w; height: grid.h; operation: CalcEngine.multiplication; }
             CalcButton { width: grid.w; height: grid.h; operation: "4"; }
             CalcButton { width: grid.w; height: grid.h; operation: "5"; }
             CalcButton { width: grid.w; height: grid.h; operation: "6"; }
-            CalcButton { width: grid.w; height: grid.h; operation: "-"; }
+            CalcButton { width: grid.w; height: grid.h; operation: "+"; }
 
+            CalcButton { width: grid.w; height: grid.h; operation: "C"  }
             CalcButton { width: grid.w; height: grid.h; operation: "1"; }
             CalcButton { width: grid.w; height: grid.h; operation: "2"; }
             CalcButton { width: grid.w; height: grid.h; operation: "3"; }
-            CalcButton { width: grid.w; height: grid.h; operation: "+"; }
+            CalcButton { width: grid.w; height: grid.h; operation: "="; }
 
+            Text { text: " " }
```

Comment:
Rather than an empty Text, I would use an empty Item {} it should work the same and be a bit more lightweight

Reply:
The item needed defined width and height to reserve space.